### PR TITLE
Sc/cherrypick 1.1.4.a

### PR DESCRIPTION
**Pull request by screret**

### CODE REVIEW

File: src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java [hunk 1]
```diff
@@ -138,7 +144,7 @@ public static void writeByteArray(ResourceLocation id, @Nullable String subdir,
     @Nullable
     @Override
     public InputStream getRootResource(String elements) {
-        throw new UnsupportedOperationException("Dynamic Resource Pack cannot have root resources");
+        return this.getClass().getResourceAsStream(elements);
```
Comment (mikerooni) [inline src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java]: Is this intentionally only in the cherrypick, or should it be in 1.20 too?

Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java]: 1.20 does it differently iirc?

Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java]: lemme test

Comment (screret) [reply src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java]: should just return null here probably...

Comment (mikerooni) [reply src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java]: Hmm, makes sense I think ^^

**Review by mikerooni — APPROVED** [review]

**Pull request merged**